xfce/ristretto/Portfile

```diff
@@ -15,12 +15,12 @@ long_description ${description}
 homepage        https://www.xfce.org
 master_sites    http://archive.xfce.org/src/apps/${name}/${branch}/
```

Comment:
Can we modify this `master_sites` URL to be HTTPS while we're updating this port?

Reply:
Done.